pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
+
+
+@pytest.fixture()
+def github():
+    """Provides the 'github' module with caching disabled"""
+    original_cache = github_module.cache
+    github_module.cache = NullCache()
+    yield github_module
```

Comment:
Btw. pytest fixtures call the code after `yield` even if the test itself fails. But of course using proper resource management (with-block or try-finally) is always a good practice.